src/chaplin/views/view.coffee

```diff
@@ -194,7 +194,7 @@ module.exports = class View extends Backbone.View
   #   e.g.
   #   @delegate('click', 'button.confirm', @confirm)
   delegate: (eventName, second, third) ->
-    if Backbone.utils
+    if Backbone.utils and Backbone.utils?.delegate
```

Comment:
> The latest release of exoskeleton does not include these functions (possibly deprecated?).

I know, I'm the one who removed them :)

The right answer is to fix Chaplin to work with NativeView and not rely on utils.delegate / utils.undelegate. I started #780 last year but I haven't had a project that needed Chaplin in a long time and so I haven't had time to dedicate to fix all the edge cases. Feel free to contribute to that pull, or just use Exoskeleton 0.6.x which should work with Chaplin.